onnx/shape_inference/implementation.cc

```diff
@@ -294,6 +295,10 @@ class ShapeInferenceImplBase {
           } else if (attr.type() == AttributeProto::SPARSE_TENSOR && attr.has_sparse_tensor()) {
             input_sparse_data_by_name[n.output(0)] = &attr.sparse_tensor();
           }
+        } else if (attr.name() == "value_ints" && attr.type() == AttributeProto::INTS) {
```

Comment:
I would put `attr.type() == AttributeProto::INTS` first. Both conditions should be equivalent but checking an int is faster than a string.